src/server/routes/index.ts

```diff
@@ -64,4 +65,7 @@ if (okta.enabled) {
 
 router.use(uncachedRoutes);
 
+// email template routes
+uncachedRoutes.use(emailTemplates);
```

Comment:
Should this go above `router.use(uncachedRoute)` ?

Reply:
Oh yes. 👍